tests/unit/test_charm.py

```diff
@@ -129,6 +273,49 @@ def test_instance_address_is_set_to_this_unit_ip(self):
         self.assertEqual(config["common"]["ring"]["instance_addr"], "1.1.1.1")
 
 
+class TestPebblePlan(unittest.TestCase):
+    """Feature: Multi-unit loki deployments.
+
+    Background: TODO
+    """
+
+    @patch("charm.KubernetesServicePatch", lambda x, y: None)
+    @patch_network_get(private_address="1.1.1.1")
+    @given(st.booleans(), st.integers(1, 3), st.integers(0, 3))
```

Comment:
@Abuelodelanada `hypothesis` is used here.
I added it to save time. Feel free to reduce this to a single test or remove completely.